server/src/internalClusterTest/java/org/elasticsearch/snapshots/SharedClusterSnapshotRestoreIT.java

```diff
@@ -1080,13 +1080,7 @@ public void testSnapshotStatus() throws Exception {
         );
 
         // Create index on 2 nodes and make sure each node has a primary by setting no replicas
-        assertAcked(
-            prepareCreate(
-                "test-idx",
-                2,
-                Settings.builder().put("number_of_replicas", 0).put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, randomIntBetween(2, 10))
-            )
-        );
+        assertAcked(prepareCreate("test-idx", 2, indexSettingsNoReplicas(randomIntBetween(2, 10))));
```

Comment:
I've seen `indexSetting(x, 0)` before so I wonder why there's a `indexSettingsNoReplicas` function just here?

Reply:
Historic reasons only, I initially just dried this stuff up for snapshot tests.
I wonder if maybe this is a pointless override though and it should instead of creating it further up the stack just be inlined ... it's not even necessarily easier to read than a `,0)`param I think :)